boards/arm/nrf52810_pca10040/Kconfig

```diff
@@ -6,4 +6,9 @@
 
 if BOARD_NRF52810_PCA10040
 
-endif # BOARD_NRF52810_PCA10040
+config BOARD_ENABLE_DCDC
+        bool "Enable DCDC mode"
+        select SOC_DCDC_NRF52X
+        default y
+
+ # BOARD_NRF52810_PCA10040
```

Comment:
Is there something wrong here, @carlescufi ?

Reply:
Looks in my diff that endif is missing

Reply:
Fixed, sorry.